### ISSUE TITLE
Doom64EX as Flatpak app

### ISSUE THREAD

**Issue by mbugni** [svkaiser/Doom64EX#117]
Hi folks,
I don't know if this project is still alive, anyway I created a [Flatpak app](https://github.com/mbugni/Doom64EXFlatpak) for Doom64EX.
Just for fun.

I built it using fluidsynth. It runs smoothly, but I get a lot of warnings like this:

```
fluidsynth: warning: Ringbuffer full, try increasing synth.polyphony! 
```

Just in case someone wants to run Doom64EX as an app.

Regards

**Comment by 7dog123** [comment]
I'm not a part of this but it hasn't been updated in two years. Here's a more up to date version [Doom64EX-Plus](https://github.com/atsb/Doom64EX-Plus)

**Comment by mbugni** [comment]
> I'm not a part of this but it hasn't been updated in two years. Here's a more up to date version [Doom64EX-Plus](https://github.com/atsb/Doom64EX-Plus)

Yes, also this one can be [built as Flatpak app](https://github.com/mbugni/Doom64EX-Plus-Flatpak).
